### ISSUE TITLE
Ignore user selected elements

### ISSUE THREAD

**Issue by anthonydillon** [kellym/smartquotes.js#51]
Could it be possible to pass a list of elements to the ignored? 

For example:
```js
options {
  'target': document.getElementById('main-content'),
  'exclude': document.querySelectorAll('.lazyload noscript'),
}
smartquotes(options);
```

This is issue arose as we use using this module which uses lazysizes to load the content from a `noscript` tag and inject it into the page to lazyload them. Here is the template logic: 
https://github.com/canonical-web-and-design/canonicalwebteam.image-template/blob/master/canonicalwebteam/templates/image_template.html

An example of the failed output:
https://codepen.io/anthonydillon/pen/PozpjRd

**Comment by kellym** [comment]
Hmm, this looks like your issue is caused from a bug in the script. I think the feature you're requesting wouldn't be necessary if it was handling `noscript` tags correctly. I'll take a look.

**Comment by anthonydillon** [comment]
Yes of course, if `noscript` was added the exclude elements list that would resolve this. I wasn't sure if you could blacklist all content within `noscript`. But, I guess it doesn't matter if there is content within the `noscript` element as the script couldn't affect that anyway.

**Comment by kellym** [comment]
This was resolved with #44. If that's incorrect, chime in here and we'll reopen.
